src/monitor/version_compat.c

```diff
@@ -1,68 +0,0 @@
-/*-------------------------------------------------------------------------
- *
- * src/monitor/version_compat.h
- *	  Compatibility macros for writing code agnostic to PostgreSQL versions
- *
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the PostgreSQL License.
- *
- *-------------------------------------------------------------------------
- */
-
-#include "postgres.h"
-
-#if (PG_VERSION_NUM < 110000)
```

Comment:
If you want I can keep the file and just drop this block.

Reply:
Let's keep the file wholesale, I think that's best (see other comment).